### DISCUSSION TITLE
Automatic escalation to Cases

### DISCUSSION THREAD

**Discussion by dgmyr** [Security-Onion-Solutions/securityonion#10074] [General]
Hi, It's possible to escalate elastalert alerts to SO Cases? 
Thanks

**Comment by cm-ops** [comment]
Are you seeing them in the Alerts tab in SOC? If so you can escalate an alert to Cases. https://docs.securityonion.net/en/latest/cases.html#creating-a-new-case

**Reply by dgmyr** [reply]
Yes, I see these alerts. I meen if it's possibile to make a custom output for elastalert rules in order to automaticly open a new case when the rules triggers. Thanks
